baseplate/lib/metrics.py

```diff
@@ -347,6 +347,8 @@ def send(self, delta: float, sample_rate: float) -> None:
         :param sample_rate: What rate this counter is sampled at. [0-1].
 
         """
+        if delta == 0:
```

Comment:
while i don't think this is really a bad change, what's the context for it happening here?

Reply:
The context is that I misunderstood your suggestion to not be sending packets if there's no data. I'm happy to take or leave it.